Task26

dkosmas opened the pull request:
Documentation and tests for idealsFromGeneratingSets. Question for all functions: show we have SeeAlso in the documentation to the optional inputs? Some functions do, some don't, so we should probably be consistent.

dkosmas committed:
idealsFromGeneratingSets tests

dkosmas committed:
update tests

dkosmas committed:
update tests

dkosmas committed:
documentation, additional test

dkosmas committed:
update documentation

dkosmas committed:
Update RandomMonomialIdeals.m2

dkosmas committed:
update documentation

dkosmas committed:
removed unnecessary SeeAlso

Sondzus commented:
@dkosmas  agreed with your question above, as we discussed yesterday, we should include SeeAlso for any method referenced by the method we are documenting, unless it is an optional input which shows up in the doc by default. 
[hint: if on Windows and can't run "viewHelp", you can still run "help" in command line and see what doc looks like! :) ]

Sondzus commented:
I am OK with this branch as it stands; someone else double-check it if you'd like (for consistency, I suppose). The package installs and checks properly.

sonjapetrovic committed:
added useful text after "end"

for copy+paste purposes ! :)

tzielin1 commented:
Installs and checks on my end. 
You use randomMonomialSets in the example, should we add it into the See also?

dkosmas commented:
@tzielin1 I'm not sure that's necessary, since it's not necessary for the function to run, nor does it call the function. We could have just made our own monomial sets in the example and used them instead of randomly generating them. But, if others think that it should be added, I'm not opposed.

sonjapetrovic committed:
small doc edits

1) removed unnecessary name for output.
2) let the user see the sets B so they can better understand this method
3) given 2, perhaps there is no reason to have N=4 and n=5 now.

Sondzus commented:
I'm good without the additional "SeeAlso". 

(FYI I made a small change - see my comments in "small doc edits" I just pushed.)

I'm good with merging this now, for real. :)

Sondzus committed:
Merge branch 'master' into task26

Commit:
Merge branch 'master' into task26

# Conflicts:
#	RandomMonomialIdeals.m2

Commit:
Merge branch 'master' into task26
